.prettierignore

```diff
@@ -16,3 +16,5 @@ packages/next-codemod/**/*.js
 packages/next-codemod/**/*.d.ts
 packages/next-env/**/*.d.ts
 test-timings.json
+test/integration/custom-routes/out/**/*.html
```

Comment:
I think a negative lookahead would be better here, `test` only holds `.js`,`.jsx`,`.ts`, and `.tsx` so it'd be useful to only match for those.